README.rst

```diff
@@ -1,186 +1,160 @@
+.. code-block:: bash
 
-The GridTools framework is a set of libraries and utilities to develop
-performance portable applications in the area of weather and climate. To
-achieve the goal of performance portability, the user-code is written in
-a generic form which is then optimized for a given architecture at
-compile-time. The core of GridTools is the stencil composition module
-which implements a DSL embedded in C++ for stencils and stencil-like
-patterns. Further, GridTools provides modules for halo exchanges,
-boundary conditions, data management and bindings to C and Fortran.
+    $ git clone https://github.com/GridTools/gt4py.git && cd gt4py
+    $ python -m venv venv
+    $ source venv/bin/activate
+    $ pip install --upgrade setuptools wheel pip
+    # For the CUDA backends add the '[cudaXX]' optional dependency
+    $ pip install -e ./[cuda11]
 
+There are notebooks in the ``examples/`` directory that can be run using IPython
+notebooks on Jupyter.
 
-Installation instructions
--------------------------
+.. code-block:: bash
 
-GT4Py contains a standard ``setup.py`` installation script that might be
-installed as usual with *pip*. Additional commands are provided to
-install and remove the GridTools C++ sources, which are not contained in
-the package.
+   $ pip install jupyterlab
+   $ jupyter-lab
 
-**IMPORTANT:** if the user provides a custom installation of GridTools
-C++ sources, it should be compatible with the latest stable release of
-GridTools, which is the version targeted by GT4Py. Note that a
-compilation problem may also appear if there is a different GridTools
-C++ version installed in a standard prefix (e.g. ``/usr/local``) which
-could be included in user-provided or standard ``setuptools`` include
-paths (for example if *Boost* is installed in the same prefix).
+There are two options to run the unit and integration tests in ``tests/``:
 
-As usual in Python, we strongly recommended to create a new virtual
-environment for any project:
+1. Use tox: ``pip install tox && tox -r -e py38-all-cpu``.
+2. Install the development requirements: ``pip install -r requirements-dev.txt``,
+   then ``pytest`` can execute the tests directly.
 
-::
 
-    # Create a virtual environment using the 'venv' module
-    python -m venv path_for_the_new_venv
+📖 Description
+--------------
 
-    # Activate the virtual environment and make sure that 'wheel' is installed
-    source path_for_the_new_venv/bin/activate
-    pip install --upgrade wheel
+GT4Py is a Python library for expressing finite-difference stencils
+related to weather and climate modeling using a high-level
+DSL using Python functions.
+These functions are compiled by the framework into high-performance
+implementations for CPUs and GPUs.
 
-Recommended installation for regular users
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The DSL expresses the stencils using the parallel model from the
+`GridTools C++ Framework <https://github.com/GridTools/gridtools>`__,
```

Comment:
The parallel model is actually slightly different: https://github.com/GridTools/concepts/blob/master/GTScript-Parallel-model.md